packages/vow/src/watch.js

```diff
@@ -156,19 +145,11 @@ export const prepareWatch = (
     /** @type {import('./types.js').VowKit<TResult1 | TResult2>} */
     const { resolver, vow } = makeVowKit();
 
-    const { promiseWatcher, vowSetter } = makeWatcherKit(resolver, watcher);
+    // Create a promise watcher to track vows, retrying on disconnection.
```

Comment:
```suggestion
    // Create a promise watcher to track vows,
    // retrying upon rejection as controlled by `rejectionMeansRetry`.
```
Also, what are the precise semantics of "retry" here? I didn't track the introduction of vows as closely as I should have, but the general mechanics of the system at large are that restarting a vat causes the kernel to settle all promises yet to be decided by it with a rejection, prompting all sufficiently-informed subscribing vats to request a _replacement_ promise from the new incarnation using an approach that varies across code responsible for producing such a promise. I think the "vow" approach replaces such exported promises with exported durable objects, so is "retry" a matter of invoking a method on a vow to get a promise replacing the rejected promise from a previous incarnation?

Reply:
That's correct.  I'll find a place to document it.